### PR TITLE
Fillt out blanks in resources and community

### PR DESCRIPTION
**Pull request by dholbach**
cc @hiddeco

### CODE REVIEW

**Review by hiddeco — APPROVED** [review]

File: config.toml [hunk 1]
```diff
@@ -120,30 +120,30 @@ With Flux, everything is controlled through pull requests, which means no learni
...
 
 [[params.resources.items]]
-title       = "Quisque aliquet libero ut ultrice"
+title       = "The Power of GitOps with Flux v2 & GitOps Toolkit with Leigh Capili"
 description = """
-Integer tortor ex, porttitor quis porttitor at, mattis et erat. Fusce in ullamcorper elit.
+Leigh shows off the Power of GitOps with Flux. This is part 1 - stay tuned for the next installment.
 """
-url         = "#"
+url         = "#https://www.youtube.com/watch?v=0v5bjysXTL8"
```
Comment (hiddeco) [inline config.toml:139]:
```suggestion
url         = "https://www.youtube.com/watch?v=0v5bjysXTL8"
```

**Review by hiddeco — CHANGES_REQUESTED** [review]

File: config.toml [hunk 2]
```diff
@@ -120,30 +120,30 @@ With Flux, everything is controlled through pull requests, which means no learni
...
 
 [[params.resources.items]]
-title       = "Quisque aliquet libero ut ultrice"
+title       = "Flux Deep Dive: A GitOps Approach to Progressive Delivery - Stefan Prodan & Hidde Beydals"
 description = """
-Integer tortor ex, porttitor quis porttitor at, mattis et erat. Fusce in ullamcorper elit.
+In their KubeCon talk, Hidde and Stefan, both maintainers of Flux, show a GitOps approach to Progressive Delivery.
 """
-url         = "#"
+url         = "https://www.youtube.com/watch?v=8v94nUkXsxU"
```
Comment (hiddeco) [inline config.toml:146]:
```suggestion
title       = " Introduction To GitOps and GitOps Toolkit"
description = """
<some description>
"""
url         = "https://www.youtube.com/watch?v=qQBtSkgl7tI"
```

Looking back, this is a better introduction than the deep dive we did on KubeCon where we tried to include too much information in 40 minutes.

**Review by hiddeco — APPROVED** [review]
Thank you @dholbach :bouquet:

**Comment by dholbach** [comment]
:small_airplane:

**Pull request merged**